meinberlin/assets/scss/components_user_facing/_alert.scss

```diff
@@ -1,26 +1,30 @@
-@use "sass:color";
+.alert {
+    margin: $padding 0;
+    padding: $padding * 0.625;
+    background-color: $message-light-blue;
+    color: $text-base;
+    text-align: start;
```

Comment:
this can go, as its the default behavior

Reply:
color and text-align and padding can go but from my point of view we need the background-color